Frameworks/CoreText/CTFrame.mm

```diff
@@ -109,12 +109,12 @@ void CTFrameDraw(CTFrameRef frame, CGContextRef ctx) {
         CGPoint relativePosition = frame->_lineOrigins[i];
 
         for (size_t j = 0; j < [line->_runs count]; ++j) {
-            _CTRun* curRun = [line->_runs objectAtIndex:j];
+			__CTRun* curRun = static_cast<__CTRun*>([line->_runs objectAtIndex:j]);
```

Comment:
since you've done the conversion of line->_runs to a CFArray, you can make these two changes to CFArray as well!